### ISSUE TITLE
Plantar ecchymosis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11154]
**Preferred term label:**
Plantar ecchymosis

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0031364

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Lisfranc injury

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001318
- label: Plantar ecchymosis
- parents: [Ecchymosis (HP:0031364)](https://hpo.jax.org/app/browse/term/HP:0031364)
- pmid: PMID:37692589
- def: A purpuric lesion that is larger than 1 cm in diameter and is located on the sole of the foot.
- comment: Plantar ecchymosis may be observed in Lisfranc Injury.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11154
- synonyms:
